<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PositionAdd</class>
 <widget class="QDialog" name="PositionAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>135</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Command</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(46, 52, 54);
color:rgb(114, 159, 207);
</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(46, 52, 54);
color:rgb(114, 159, 207);
</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="auto_direction_button">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">#auto_direction_button
{
	border-image: url(:/images/toggle_button_off.png);
}

#auto_direction_button:checked
{
border-image: url(:/images/toggle_button_on.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(114, 159, 207);</string>
   </property>
   <property name="text">
    <string>Auto Direction</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>80</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="position">
    <widget class="QLineEdit" name="x_position">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(114, 159, 207);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="y_position">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(114, 159, 207);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(114, 159, 207);</string>
     </property>
     <property name="text">
      <string>Step2 Position</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(114, 159, 207);</string>
     </property>
     <property name="text">
      <string>Step1 Position</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="direction">
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(114, 159, 207);</string>
     </property>
     <property name="text">
      <string>Direction</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(114, 159, 207);</string>
     </property>
     <property name="text">
      <string>Duration</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="duration_command">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(114, 159, 207);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="direction_command">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(114, 159, 207);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PositionAdd</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PositionAdd</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
